keyboards/dztech/dz65rgb/keymaps/jumper149/rgb_matrix_user.inc

```diff
@@ -0,0 +1,700 @@
+/* Copyright 2021 Felix Springer
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 2 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+RGB_MATRIX_EFFECT(base_effect)
+RGB_MATRIX_EFFECT(base_ad_effect)
+RGB_MATRIX_EFFECT(vi_effect)
+RGB_MATRIX_EFFECT(fn_indicator)
+RGB_MATRIX_EFFECT(kb_indicator)
+RGB_MATRIX_EFFECT(kitt_effect)
```

Comment:
For reference, these will be translated to: 
```
RGB_MATRIX_CUSTOM_base_effect
RGB_MATRIX_CUSTOM_base_ad_effect
RGB_MATRIX_CUSTOM_vi_effect
RGB_MATRIX_CUSTOM_fn_indicator
RGB_MATRIX_CUSTOM_kb_indicator
RGB_MATRIX_CUSTOM_kitt_effect
```

Reply:
This is the help I didn't ask for, but that I appreciate a lot! :)